common/util/file_util.cc

```diff
@@ -46,6 +48,46 @@ absl::string_view Stem(absl::string_view filename) {
              : filename.substr(0, last_dot_pos);
 }
 
+// TODO (after bump to c++17) rewrite this function to use std::filesystem
+bool UpwardFileSearch(absl::string_view start, absl::string_view filename,
+                      std::string* result) {
+  static const std::string dir_separator = "/";
```

Comment:
or `constexpr absl::string_view` (no allocation)

Reply:
Thanks for the suggestions, changed